src/coreComponents/constitutive/solid/Damage.hpp

```diff
@@ -160,6 +232,13 @@ class DamageUpdates : public UPDATE_BASE
     return m_strainEnergyDensity( k, q );
```

Comment:
I think this function is returning the maximum of the strainEnergyDensity over time. Although this is good to enforce irreversibility in standard phase-field models, it might negatively impact the nucleation model as it needs healing of the damage field. (I don't know how much of an impact this may have, how are our benchmarks when compared to Kumar's paper?)